content/blog/posts/mps-blog.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "Explore Multiplayer Gaming with the O3DE Multiplayer Sample - Now Available for Download!"
+date: 2055-04-19
+slug: mps-sample 
+Author: Darrin McPherson
+blog_img: "/images/blog/mps-sample/image01.png"
+full_img: "/images/blog/mps-sample/image01.png"
+---
+
+We are happy to announce the release of the O3DE Multiplayer Sample Game (MPS). The MPS is a fully open source mini game where players battle one another for dominance in a multi-tiered, under-construction, Starbase. This new sample game is intended to serve as an example for the O3DE community to use in the creation of other games as well as to demonstrate a number of features in O3DE, including networking, material canvas, asset creation best practices, and the setup of visual effects setup using the [PopcornFX gem](https://github.com/PopcornFX/O3DEPopcornFXPlugin), and audio effects using the [WWise gem](https://github.com/o3de/o3de/tree/development/Gems/AudioEngineWwise). 
+
+The design intent of the MPS was to create something that would be fun to play, feature exciting exploration that rewarded player mastery, have clear risk vs. reward elements, and feature both player vs. environment, and player vs. player gameplay. We also wanted it to be highly configurable and expandable so that the O3DE community continue to iterate and add to it. From a technical angle, we also wanted a multi-faceted game that would provide a platform to test against when adding new systems to O3DE.
+
+It is best to think of the MPS as a starting point and a toolbox of new features. We encourage you to get creative in how you expand it, adjust it or use its various features to create wholly different games. We look forward to seeing what you make of it!
+
+## Partners & License Terms of Use
+
+This sample was created through a partnership between AWS Gametech, Adobe, KitBash3D and the O3DF. 
+
+* The use of the Y-bot character model as well as the animations contained in character_mps gem are subject to the license terms found here:  https://github.com/o3de/o3de-multiplayersample-assets/tree/main/Gems/character_mps
+* The use of Kitbash3D High Tech Streets Kit are subject to the license terms found [here](https://github.com/o3de/o3de-multiplayersample-assets/tree/main/Gems/kb3d_mps)
+
+## Installing and Running the O3DE Multiplayer Sample Game
+
+In order to setup and build the MPS you will need to get follow the setup instructions for these two repositories:
```

Comment:
Maybe highlight the MPS README as a link as it has full instructions for Linux and Windows?

If this goes out while stabilization is live, then technically the head/active branch is stablization/2305.

Reply:
agreed!